modules/_node-scripts/tsconfig/writeProjectTsconfig.mjs

```diff
@@ -90,11 +92,26 @@ export default async function writeProjectTsconfig(
 		references,
 	};
 
+	json['@generated'] = hash(json);
```

Comment:
this isn't totally necessary, but at least gives us a way to track it's history. Or we can compare hashes before writing files if we think that is necessary